internal/util/logger/logger.go

```diff
@@ -74,22 +75,24 @@ type LogSink struct {
 
 // New returns a new NumaLogger with a logr.Logger instance with a
 // default setup for zerolog, os.Stdout writer, and info level.
-func New() NumaLogger {
+func New() *NumaLogger {
 	w := io.Writer(os.Stdout)
 
-	lvlStr := os.Getenv(common.EnvLogLevel)
```

Comment:
Is there any reason why we need this environment variable? Seems better to just have one place (the ConfigMap) for storing it